src/test/resources/jsonld/HT015891797

```diff
@@ -74,13 +74,6 @@
   } ],
   "hbzId" : "HT015891797",
   "id" : "http://lobid.org/resources/HT015891797#!",
-  "isPartOf" : [ {
-    "id" : "http://lobid.org/resources/HT015845132#!",
-    "label" : "lobid Ressource"
-  }, {
-    "id" : "http://lobid.org/resources/HT002786685#!",
-    "label" : "lobid Ressource"
-  } ],
```

Comment:
Is it correct to loose the link to HT002786685 ? Can't find it redundantly residing in an other field.